docs/openapi.yaml

```diff
@@ -1299,7 +1308,7 @@ paths:
             default: false
         - name: until_block
           in: query
-          description: returned data representing the state up until that point in time, rather than the current block.
+          description: returned data representing the state up until that point in time, rather than the current block. Use either of unachored or until_block parameters.
```

Comment:
```suggestion
          description: returned data representing the state up until that point in time, rather than the current block. Use either of `unachored` or `until_block` parameters.
```